src/igvfd/schemas/changelogs/human_donor.md

```diff
@@ -3,3 +3,4 @@
 ### Minor changes since version 1
```

Comment:
Can add an extra blank line was inserted after this header to match the changes in the other changelogs